_includes/macros.html

```diff
@@ -239,8 +246,6 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
     </div>
 {%- endmacro %}
 
-
-
```

Comment:
There probably shouldn't have been 3, but let's try to keep 2 blank lines between macros, since single blank lines are used liberally within each macro.